docs/configuration-ini.md

```diff
@@ -53,227 +53,472 @@ Below is sample Network section.
-
-   Initial learning rate to decay
+2. ```beta1 = <float>```
 
-3. ```beta1 = <float>```
+   beta1 parameter for adam optimizer. Only valid for adam. The default value is 0.9.
 
-   beta1 parameter for adam optimizer. Only valid for adam.   0.9 is default.
 
-4. ```beta2 = <float>```
+3. ```beta2 = <float>```
 
-   beta2 parameter for adam optimizer. Only valid for adam. 0.999 is default.
+   beta2 parameter for adam optimizer. Only valid for adam. The default value is 0.999.
 
-5. ```epsilon = <float>```
+4. ```epsilon = <float>```
 
-     Epsilon parameter for adam optimizer. Only valid for adam. 1.0e-7 is default.
+   Epsilon parameter for adam optimizer. Only valid for adam. The default value is 1.0e-7.
 
 Below is a sample Optimizer section.
 
 ```ini
 # Optimizer Section
 [Optimizer]
-Type = adam
-Learning_rate = 1e-4
-batch_size = 32
+type = adam
 beta1 = 0.9
 beta2 = 0.999
 epsilon = 1e-7
 ```
 
-### DataSet Section
-
-Define data set. training, validation, test data set.
-
-Start with "[ DataSet ]"
-
-1. ```buffersize = <unsigned int>```
-
-    Define Buffer size. usually it is greater than batch size.
-
-    Data buffer thread keeps reading the data from the file and stores the data into the data buffer.
-    Meanwhile main thread gets the training data from this data buffer and feeds it to the model.
-    This keyword defines the size of Data Buffer.
-
-2. ```traindata = <string>```
-
-    training data file path.   The data must be saved as following
-
-    ```feature data[i], label data[i], feature data[i+1], label data[i+1], ...```
-
-3. ```validdata = <string>```
-
-    validation data file path.   The data must be saved as following
-
-    ```feature data[i], label data[i], feature data[i+1], label data[i+1], ...```
-
-4. ```testdata = <string>```
-
-    test data file path.   The data must be saved as following
-
-    ```feature data[i], label data[i], feature data[i+1], label data[i+1], ...```
-
-5. ```labeldata = <string>```
+### Learning Rate Scheduler Section
 
-    label data file path. The data must be saved as following
+Define the type, learning rate, decay steps and decay rate.
 
-    ```class Name [i], class name [i+1],...```
-
-### Layer Section
-
-Describe hyper-parameters for layer. Order of layers in the model follows the order of definition of layers here from top to bottom.
-
-Start with "[ ${layer name} ]". This layer name must be unique throughout network model.
+Start with "[LearningRateScheduler]"
 
 1. ```type = <string>```
+   constant, exponential and step are supported.
+      * constant : constant learning rate
+      * exponential : exponential decay
+      * step: step decay
 
-   Type of Layer
-     * input : input layer
-     * conv2d : 2D convolution layer
-     * pooling2d : 2D pooling layer
-     * flatten : flatten layer
-     * fully_connected : fully connected layer
-     * batch_normalization : batch normalization layer
-     * activation : activation layer
+2. ```learning_rate = <float>```
```

Comment:
1. learning_rate property can be a vector of float value in step scheduler.
2. Iteration property is missing. This property is used in step scheduler

Reply:
What kind of type for iteration ?

Reply:
The type of iteration is unsigend int.
The step scheduler is added recently so there is no unit test for that for now.

Reply:
okay thank you.